lib/plugins.js

```diff
@@ -686,24 +603,26 @@ function plugins (settings, deviceId) {
 
   function checkCryptoBalance (fiatCode, rec) {
     const [cryptoCode, fiatBalance] = rec
-
     if (!fiatBalance) return null
 
     const notifications = configManager.getNotifications(cryptoCode, null, settings.config)
-    const lowAlertThreshold = notifications.cryptoLowBalance
-    const highAlertThreshold = notifications.cryptoHighBalance
+    const override = _.find(override => override.cryptoCurrency === cryptoCode, settings.config.notifications_cryptoBalanceOverrides)
+    const lowAlertThreshold = override ? override.lowBalance : notifications.cryptoLowBalance
+    const highAlertThreshold = override ? override.highBalance : notifications.cryptoHighBalance
```

Comment:
I'm pretty sure that this code is redundant, getNotifications passing down cryptoCode should already be resolving the overrides for you.